README.md

```diff
@@ -20,91 +27,98 @@ This code is dual licensed under the MIT license and the OGL (Open Government Li
 The contents of this repository are protected by Crown Copyright (C).
 
 ## Development
+It is recommended that you use visual studio code and a devcontainer as this will install all necessary components and correct versions of tools and languages.  
+See https://code.visualstudio.com/docs/devcontainers/containers for details on how to set this up on your host machine.  
 
-### Requirements
-* make
-* nodejs + npm/yarn
-* [poetry](https://github.com/python-poetry/poetry)
-* Java 8+
 
-### Install
-```
-$ asdf install
-$ make install
-```
+All commits must be made using [signed commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)
 
-#### Updating hooks
-You can install some pre-commit hooks to ensure you can't commit invalid spec changes by accident. These are also run
-in CI, but it's useful to run them locally too.
+Once the steps at the link above have been completed. Add to your ~/.gnupg/gpg.conf as below:
 
 ```
-$ make install-hooks
+use-agent
+pinentry-mode loopback
 ```
 
-### Environment Variables
-Various scripts and commands rely on environment variables being set. These are documented with the commands.
-
-:bulb: Consider using [direnv](https://direnv.net/) to manage your environment variables during development and maintaining your own `.envrc` file - the values of these variables will be specific to you and/or sensitive.
-
-### Make commands
-There are `make` commands that alias some of this functionality:
- * `lint` -- Lints the spec and code
- * `publish` -- Outputs the specification as a **single file** into the `build/` directory
- * `serve` -- Serves a preview of the specification in human-readable format
-
-### Testing
-Each API and team is unique. We encourage you to use a `test/` folder in the root of the project, and use whatever testing frameworks or apps your team feels comfortable with. It is important that the URL your test points to be configurable. We have included some stubs in the Makefile for running tests.
+and to your ~/.gnupg/gpg-agent.conf as below:
 
-### VS Code Plugins
+```
+allow-loopback-pinentry
+```
 
- * [openapi-lint](https://marketplace.visualstudio.com/items?itemName=mermade.openapi-lint) resolves links and validates entire spec with the 'OpenAPI Resolve and Validate' command
- * [OpenAPI (Swagger) Editor](https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi) provides sidebar navigation
+As described here:
+https://stackoverflow.com/a/59170001
```

Comment:
Maybe a bit about the GPG credential caching too?
https://superuser.com/questions/624343/keep-gnupg-credentials-cached-for-entire-user-session

Reply:
fixed (and added to other repo)